snap/snapcraft.yaml

```diff
@@ -1,36 +1,87 @@
-name: saber
-version: 0.12.10
-summary: Handwritten notes
-description: A cross-platform libre handwritten notes app, available for Android, iOS, Linux, Windows, and macOS!
-
-confinement: strict
-base: core22
-grade: devel
-
-slots:
-  dbus-saber:
-    interface: dbus
-    bus: session
-    name: com.adilhanney.saber
-
-apps:
-  saber:
-    command: saber
-    extensions: [flutter-stable]
-    plugs:
-      - network
-    slots:
-      - dbus-saber
-
+name: saber # you probably want to 'snapcraft register <name>'
+base: core22 # the base snap is the execution environment for this snap
+adopt-info: saber
+grade: stable # must be 'stable' to release into candidate/stable channels
+confinement: strict # use 'strict' once you have the right plugs and slots
+license: GPL-3.0
+compression: lzo
+icon: icon.png
```

Comment:
I see that you've deleted `snap/gui/saber.png`. Where does this `icon.png` come from?

Reply:
Hi thanks for pointing it out. I forgot to update the path. This is the new updated path of icon.

Reply:
Weren't you using a workflow for the snap builds in your old repo, or was that just handled by snap?

Reply:
No, it's there just to have it tested. Nothing extra. It's done by connecting this github repo directly to the snapcraft builder in the `Builds` option of the [`Snapcraft Builder`](https://snapcraft.io/saber/builds)

Reply:
Ah okay, yeah a workflow would be a nice assurance :)